README.md

```diff
@@ -110,12 +86,18 @@ To run the project, execute the following command:
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Mert Kaya**
+👤 **Bashir Yussuf**
 
 - GitHub: [@yussufOB](https://github.com/yussufOB)
 - Twitter: [@_ybash](https://twitter.com/_ybash)
 - LinkedIn: [yussuf-Bashir](https://www.linkedin.com/in/yussuf-bashir/)
 
+👤 **Fuad Nabiyev**
+
+- GitHub: [@githubhandle](https://github.com/FuadNabi)
+- Twitter: [@twitterhandle](https://twitter.com/FuadNebiyev2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fuad-nabiyev/)
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
Great job team 🤝

- [ ] [Optional] You may kindly fix the broken MIT license link to make your readme file more professional.